testing/integration-tests/src/utils/wait_for_blocks.rs

```diff
@@ -11,4 +11,11 @@ pub async fn wait_for_blocks<C: Config>(api: &impl OnlineClientT<C>) {
     let mut sub = api.rpc().subscribe_all_block_headers().await.unwrap();
     sub.next().await;
     sub.next().await;
+
+    #[cfg(feature = "unstable-light-client")]
```

Comment:
Nit; maybe tweak the comment to note that if unstable-light-client is enabled it'll wait for 3 more blocks, because presumably that helps it work faster? :)